operators/multiclusterobservability/api/v1beta2/multiclusterobservability_types.go

```diff
@@ -16,6 +16,18 @@ type MultiClusterObservabilitySpec struct {
 	// Advanced configurations for observability
 	// +optional
 	AdvancedConfig *AdvancedConfig `json:"advanced,omitempty"`
+	// Size read path of your Observability instance
+	// +optional
```

Comment:
optional has no effect if you don't have the `omitempty` tag.

Reply:
Thanks! Wasn't aware!